server/src/test/resources/csars/yaml/valid/lamp-csar_NoInput/template.yaml

```diff
@@ -0,0 +1,78 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+description: Template for deploying a LAMP stack.
+metadata:
+  template_name: lamp-stack-template
+  template_author: stupro-toscana
+  template_version: 1.0
+
+
+topology_template:
+  node_templates:
+    my_app:
+      type: tosca.nodes.WebApplication
+      requirements:
+        - host: apache_web_server
+        - database_endpoint: my_db
+      interfaces:
+        Standard:
+          create: my_app/install_php.sh
+          start:
+            implementation:
+              primary: my_app/start_myphpapp.sh
+              dependencies:
+                - myphpapp.php
+                - mysql-credentials.php
+
+    apache_web_server:
+      type: tosca.nodes.WebServer.Apache
+      requirements:
+        - host: server
+      interfaces:
+        Standard:
+          create: apache_web_server/webserver_install.sh
+          start: apache_web_server/webserver_start.sh
+
+    my_db:
+      type: tosca.nodes.Database.MySQL
+      properties:
+        name: { mydb }
+        user: { root }
+        password: { abc }
+        port: { 3306 }
+      capabilities:
+        database_endpoint:
+          properties:
+            port: { 3306 }
+      requirements:
+        - host: mysql_dbms
+
+    mysql_dbms:
+      type: tosca.nodes.DBMS.MySQL
+      properties:
+          root_password: { abc }
+          port: { 3306 }
+        requirements:
+          - host: server
+        interfaces:
+          Standard:
+            create: mysql_dbms/mysql_dbms_install.sh
```

Comment:
these are obsolete i guess.

Reply:
I think we need the install.sh one since the password is hardcoded here